services/src/api/handlers/datasets.rs

```diff
@@ -1226,7 +1226,7 @@ mod tests {
                 "symbology": null
             },{
                 "id": id2,
-                "name": "My_Dataset2",
+                "name": "My Dataset2",
```

Comment:
i think names are not allowed to contain " " (spaces). Only DisplayNames may contain them.

Reply:
da37fa54d75953a358c9ff55cec07e4905b4b346